ts/handler_utils/torch_export/load_model.py

```diff
@@ -0,0 +1,27 @@
+import tempfile
+
+import torch.fx._pytree as fx_pytree
+from torch._inductor.utils import aot_inductor_launcher, cache_dir
+from torch.utils import _pytree as pytree
+from torch.utils.cpp_extension import load_inline
+
+
+def load_exported_model(model_so_path, device):
+    module = load_inline(
+        name="aot_inductor",
+        cpp_sources=[aot_inductor_launcher(model_so_path, device)],
+        # use a unique build directory to avoid test interference
+        build_directory=tempfile.mkdtemp(dir=cache_dir()),
```

Comment:
does this use the same cache_dir as teh environment variable a user would be setting?

Reply:
This seems like `TORCHINDUCTOR_CACHE_DIR`